pipeline/astro_cal.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, **kwargs):
         self.override(kwargs)
 
     def get_process_name(self):
-        return 'astro_cal'
+        return 'extraction'
```

Comment:
I guess I'm not fully clear on all the implications of this.  It's weird to call this 'extraction', given that there's also a 'extraction' process in `detection.py` that really is extraction.

How is `get_process_name()` actually used?  What is this process name for?

It's used in constructing provenances, isn't it?  If so, this has become a little bit tangled... the provenance for extraction now includes all of extraction, wcs, and zp; since the latter two aren't separate steps, they don't get their own provenances, but their parmeters are folded into the extraction provenance.  But, what's the implication of the process_name here not just being astro_cal?

It seems that originally there was a mapping between one of these thingies that had a Parameters subclass and one of the steps of the pipeline, whereas that's not true any more.

Reply:
yes, I guess the get_process_name really was more useful when it was one to one between process and Parameters object. 
I will have another look and maybe we don't actually need this function anymore.

Reply:
I left the function in there, but detached it from the provenances. It is still a useful thing to have for stuff like injecting warnings and having them tagged with an informative "process" name.